config/initializers/sentry.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+if Rails.env.production?
+  Raven.configure do |config|
+    config.dsn = ENV.fetch('SENTRY_DSN', '')
```

Comment:
We're going to need a different env var; we currently have `SENTRY_DSN` for the frontend project.  Can we change both of them to `SENTRY_DSN_FRONTEND` & `SENTRY_DSN_BACKEND`?

Reply:
We have `REACT_APP_SENTRY_DSN` on the frontend but I made the change anyway.

Reply:
Hm.  My ENV VARS might have gotten out of date.